make.sh

```diff
@@ -3,21 +3,42 @@
 # Unicorn Engine
 # By Nguyen Anh Quynh <aquynh@gmail.com>, 2015
 
-MAKE_JOBS=$((${MAKE_JOBS}+0))
+usage() {
+    cat 1>&2 <<EOF
+make.sh - The build script for unicorn engine
+USAGE:
+    $ ./make.sh [OPTIONS]
+OPTIONS:
+                            Build the project
+    asan                    Build for ASan
+    install                 Install the project
+    uninstall               Uninstall the project
+    macos-universal         Build universal binaries on macOS
```

Comment:
I don't know if this description is correct or not.

Reply:
yes this is correct